tests/microos/workloads/x11-container/x11_client.pm

```diff
@@ -0,0 +1,90 @@
+# SUSE"s openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: podman x11-container
+# Summary: install and verify x11 container.
+# Maintainer: Grace Wang <grace.wang@suse.com>
+
+use warnings;
+use strict;
+use testapi;
+use lockapi;
+use base 'x11test';
+use mmapi;
+use mm_tests;
+use x11utils 'turn_off_gnome_screensaver';
+
+# MM network check: try to ping the gateway, and the server
+sub ensure_server_reachable {
+    assert_script_run('ping -c 1 10.0.2.2');
+    assert_script_run('ping -c 1 10.0.2.101');
+}
+
+sub run {
+
+    my $self = shift;
+
+    # Setup static NETWORK
+    $self->configure_static_ip_nm('10.0.2.102/24');
+
+    x11_start_program('xterm');
+    turn_off_gnome_screensaver;
+
+    mutex_wait("x11_container_ready");
+    ensure_server_reachable();
+
+    # ssh login to the x11 server
+    enter_cmd("ssh root\@10.0.2.101");
+    assert_screen 'ssh-login', 60;
+    enter_cmd "yes";
+    assert_screen 'password-prompt', 60;
+    type_password();
+    send_key "ret";
+    assert_screen 'ssh-login-ok';
+
+    # start pulseaudio container
+    my $pacontainerpath = 'registry.suse.de/suse/sle-15-sp6/update/cr/totest/images/suse/pulseaudio:17';
```

Comment:
Shouldn't we use a variable with a default here?

We have one already documented:

```suggestion
    my $pacontainerpath = get_var('CONTAINER_IMAGE_TO_TEST', 'registry.suse.de/suse/sle-15-sp6/update/cr/totest/images/suse/pulseaudio:17');
```

Reply:
> In terms of the pulseaudio and firefox kiosk container, they are in the x11_client.pm, one variable `CONTAINER_IMAGE_TO_TEST` is not enough. My question is can I define another variable such as `CONTAINER_IMAGE_TO_TEST_1` ?

I don't have an easy answer to it right now.  Perhaps specify multiple images separated by space?